src/configuration/services/mongodb.md

```diff
@@ -62,16 +62,33 @@ foreach ($relationships['database'] as $endpoint) {
   if (empty($endpoint['query']['is_master'])) {
     continue;
   }
-  $container->setParameter('mongodb_server', $endpoint['scheme'] . '://' . $endpoint['host'] . ':' . $endpoint['port']);
+  $container->setParameter('mongodb_server', $endpoint['scheme'] . '://' . $endpoint['ip'] . ':' . $endpoint['port']);
```

Comment:
why use the `ip`?

Reply:
'host' appears not to work in some configurations, whereas IP is always available, AFAIK.